storage/innobase/mtr/mtr0mtr.cc

```diff
@@ -37,6 +37,24 @@ Created 11/26/1995 Heikki Tuuri
 #include "srv0start.h"
 #include "log.h"
 #include "mariadb_stats.h"
+#include "my_cpu.h"
+
+std::pair<lsn_t,mtr_t::page_flush_ahead> (*mtr_t::finisher)(mtr_t *, size_t);
+unsigned mtr_t::spin_wait_delay;
+
+void mtr_t::finisher_update()
+{
+  ut_ad(log_sys.latch_have_wr());
+  finisher=
```

Comment:
this is a bit terse because of excessive use of ternary operator.

Reply:
Sorry, I am a fan of the ternary operator. At least I put some parentheses around it to make it a little clearer.